cmd/wallet-sdk-gomobile/openid4ci/resolveopts.go

```diff
@@ -24,7 +24,7 @@ import (
 // We don't check here to see if multiple conflicting options are used - that's left up to the
 // goapicredentialschema.Resolve method that get called after this function returns.
 func prepareOpts(credentials *Credentials, issuerMetadata *IssuerMetadata,
-	preferredLocale string,
+	preferredLocale string, logger api.Logger,
```

Comment:
no other better way to provide logger instead of injecting it manually to each and every API?

Reply:
I could make it a global variable that the caller can set in one place. However, I'm not entirely sure how the lifetime of that global variable will work when used in bindings (i.e. I'm concerned that it might become "unset" if certain things happen in an app using the SDK, like suspending it, etc)

Reply:
Also, the one you commented on is an internal function - the caller only has to pass in a logger at the beginning of the OpenID4CI and OpenID4VP interactions - (or, if they want to call the Resolve function outside of the flow)